inventories/operator-inventory.yaml

```diff
@@ -14,24 +14,44 @@ images:
     platform: linux/amd64
 
     stages:
-      - name: operator-template-ubi
+#
+# Dev build stages
+#
+      - name: operator-builder-dev
+        task_type: docker_build
+        tags: [ "ubi" ]
+        dockerfile: scripts/dev/templates/operator/Dockerfile.builder
+
+        buildargs:
+          builder_image: $(inputs.params.builder_image)
+
+        labels:
+          quay.expires-after: 48h
+
+        output:
+        - registry: $(inputs.params.registry)/$(inputs.params.operator_image_dev)
+          tag: $(inputs.params.version_id)-context
+
+      - name: operator-template-dev
         task_type: dockerfile_template
         tags: ["ubi"]
-        distro: operator
-
+        template_file_extension: operator
         inputs:
-          - builder
-          - builder_image
           - base_image
 
         output:
-          - dockerfile: scripts/dev/templates/Dockerfile.ubi-$(inputs.params.version_id)
+          - dockerfile: scripts/dev/templates/operator/Dockerfile.operator-$(inputs.params.version_id)
```

Comment:
So it seems we're now only using UBI images. If that is the case, would you mind to mention this in the release notes?

Reply:
It hasn't changed. We haven't built ubuntu in the community at all.